static/src/deploys-radiator/app/main.js

```diff
@@ -34,14 +34,39 @@ const run = () => {
         'rss',
         'sport'
     ]);
-    const filterWhitelisted = (deploys) => deploys
+    const unifiedWhitelist = List([
+        'all'
+    ]);
+    const filterServerWhitelisted = (deploys) => deploys
         .filter(deploy => serverWhitelist.contains(deploy.projectName))
         .toList();
+    const filterUnifiedWhitelisted = (deploys) => deploys
+        .filter(deploy => unifiedWhitelist.contains(deploy.projectName))
+        .toList();
```

Comment:
Any reason not to combine these? (same goes for `serverTime` below) e.g.    

    const filterWhiteListed = (whitelist) => (deploys) => deploys
        .filter(...)
        .toList()

    const filterServerWhiteListed = filterWhiteListed(serverWhiteList)